typescript/packages/subsurface-viewer/src/storybook/layers/Axes2DLayer.stories.tsx

```diff
@@ -49,6 +49,33 @@ export const Base: StoryObj<typeof SubsurfaceViewer> = {
     },
 };
 
+function MakeLabel(a: number): string {
```

Comment:
Stick to conventional (camel?) case. Can we enforce this with an eslint rule?

Reply:
I renamed the function. I may be possible to enforce camel case with eslint. But perhaps this is better done in a separate PR as it may trigger cleaup all over the place.